### PR TITLE
[1 of n: Icons] Remove inline SVG requirements

### PR DESCRIPTION
**Pull request by jfurrow**
This PR removes the project requirements for inlining SVGs. We are going to be using SVG sprites which will significantly reduce our bundled Javascript file size.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

**Comment by jfurrow** [comment]
retest this please

**Comment by jfurrow** [comment]
Integration tests passing locally:

File: jest/preprocessor.js [hunk 1]
```diff
@@ -14,18 +14,6 @@ module.exports = {
...
-        // import IconFoo from '../../icons/icon-medium/pages-code.svg?name=foo';
-        var matches = src.match(/^(\w+) (.*?) (\w+|=) '(.*?)(icons)(.*?)\.svg(\?.*?)';$/img);
-        if (matches && matches.length) {
-          for (var i = 0; i < matches.length; i++) {
-            var importMatch = matches[i].match(/^(\w+) (\w+)/);
-            // It'll replace all matches with:
-            // var IconFoo = React.createClass(...);
-            src = src.replace(matches[i], 'var ' + importMatch[2] + ' = React.createClass({render: function () { return React.createElement("div", null); } });');
-          }
-        }
```
Comment (rcorral) [inline jest/preprocessor.js]: RIP

**Pull request merged**